trainingRN/src/screens/BookList/styles.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+import colors from '../../constants/colors';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: colors.aliceBlue
+  },
+  book: {
+    flexDirection: 'row',
+    marginBottom: 10
+  },
+  image: {
+    height: 50,
+    width: 50,
```

Comment:
I'd extract this `50` within this file since this image will always behave as a square. What do you think ?

Reply:
So you are suggesting to apply the width and height directly to the Image component?